active/ListViewGridView/listviewgridview-designupdates.md

```diff
@@ -0,0 +1,506 @@
+
+`public Brush SelectedPressedBackground { get; set; }`
+
+## Detailed Task Breakdown for styling implementation
+
+1. GridView visual improvements
+
+    a. Items have rounded corners at rest.
+
+    b. Items have a new rounded inner white border when in selected states (including multiple and single selection modes).
+
+    c. In all non-rest states, items have a rounded grey backplate. For Selected and Pressed states, backplate slightly darker than hover backplate. 
+
+    d. In hover state specifically: Items have a rounded, light grey border and a backplate with a light grey fill (same color as ListView).
+
+    e. Existing accent color border that’s displayed in selected states will have rounded corners to match items.
+
+    f. Focus rect will need to be rounded to adhere to shape of backplate.
+
+2. ListView visual improvement
+
+    a. A rounded gray backplate will be added within the footprint of the item, with 2px top/bottom  and 4px left/right margin from the overall footprint on all sides. This backplate will appear in all non-rest states.
+
+    b. An accent color selection indicator visual will appear left-aligned to the backplate with no left margin when an item is selected/clicked in any state. 
+
+    c. Items will need a 2px top/bottom margin to be distinct from one another. This margin should be taken from the backplate. The overall footprint of item shouldn’t change. 
+
+    d. Focus rect will need to be rounded to adhere to backplate rather than overall footprint.
+
+    e. APIs will be added to support the new selection indicator and backplate. See API notes for more details.
+
+3. Checkbox visual changes for ListView and GridView
+
+    a. Round and update checkbox visuals used for multiple selection mode in ListView and GridView so that they match the current styling and behavior of CheckBox control (in terms of color, font, shape, animation).
+
+    b. GridView checkbox should have rounded corners a dark grey border.
+
+    c. On pressed and disabled states, GridView checkbox border color lightens. 
+
+4. Inset backplate interaction for ListView controls
+
+    a. ListView items will now have an inset rounded backplate that is smaller than their overall footprint. The overall footprint should still be clickable and have the same interaction as if you’d clicked on the inset backplate.
+
+5. Add animation for selection indicator
+
+    a. Add animations for reveal/dismissal of selection indicator for ListView controls. The animation will consist of the indicator going from invisible -> small -> full size upon reveal, and full size -> smaller -> invisible upon dismissal. This animation will apply to both single and extended selection modes if applicable.
+
+6. Miscellaneous
+
+    a. Implement global resources that allow the developer to un-round any/all GridView or ListView item corners, similar to the ControlCornerRadius resource that currently exists but does not directly affect ListView/GridViewItems.
```

Comment:
Small compat risk that an app put something important in the corner pixel, which is now clipped out. For example, Minesweeper has a cheat mode that changes the color of the upper left pixel depending on whether the mouse is hovering over a hidden mine.

Reply:
This should show up only for apps that consume the updated nuget package, so existing apps should not break.